File: 2021/Day08_n/1.py
def main(file_name):
    f = open(file_name)
    lines = f.readlines()

    total = 0
    for line in lines:
        output = line.split(" | ")[1][:-1].split(' ')
        for digit in output:
            length = len(digit)
            if length == 2 or length == 4 or length == 3 or length == 7:
                total += 1

    f.close()
    return total


print("Total times digits 1, 4, 7, 8:", main("input.txt"))

File: 2022/Day01/1.js
const fs = require("fs");

const input = fs.readFileSync("input.txt", "utf-8");
const lines = input.split(/\r?\n/);

let calories = 0;
let max_calories = 0;

for (let line of lines) {
    if (line.length === 0) {
        if (calories > max_calories) max_calories = calories;
        calories = 0
        continue
    }

    let c = parseInt(line)
    calories += c
}

console.log(max_calories)